src/style.css

```diff
@@ -0,0 +1,66 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./Avenir-Book.ttf);
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.search-bar {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-weight: 300;
+  font-family: inherit;
+}
+
+.search-form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.big-search-bar {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.small-search-bar {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.big-search-img {
+  top: 25px;
+  left: 26px;
+}
+
+.small-search-img {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.search-img {
+  z-index: 1;
+  position: absolute;
```

Comment:
position is a property, which should be written for each element separately so that you can easily understand why your block behaves like so